From 4f39dcd3553fb2ed614e5124216b9c860eb1dcde Mon Sep 17 00:00:00 2001
From: Tobias Schaffner <tobias.schaffner@siemens.com>
Date: Fri, 18 Aug 2023 11:41:05 +0200
Subject: [PATCH] visionfive2: Allow to overwrite DTB_FILES

The name of the DTB may differ if an alternative kernel is used.

Signed-off-by: Tobias Schaffner <tobias.schaffner@siemens.com>
---
 meta-isar/conf/machine/starfive-visionfive2.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-isar/conf/machine/starfive-visionfive2.conf b/meta-isar/conf/machine/starfive-visionfive2.conf
index c0f61f46..8003ec5d 100644
--- a/meta-isar/conf/machine/starfive-visionfive2.conf
+++ b/meta-isar/conf/machine/starfive-visionfive2.conf
@@ -16,4 +16,4 @@ IMAGER_BUILD_DEPS += "u-boot-starfive-visionfive2"
 
 # DTB file for booting
 DTB_VISIONFIVE2 = "jh7110-visionfive-v2.dtb"
-DTB_FILES = "starfive/${DTB_VISIONFIVE2}"
+DTB_FILES ?= "starfive/${DTB_VISIONFIVE2}"
-- 
2.34.1

